.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -112,7 +112,7 @@ body:
     attributes:
       label: Additional Relevant Configuration
       description: |
-        If your issue involves other programs, tools, or applications in addition to Ghostty (e.g. Neovim, tmux, Zellij, etc.), please provide any relevant configuration information here.  If you use custom CSS or shaders for Ghostty, also include them here, if applicable to your issue.
+        If your issue involves other programs, tools, or applications in addition to Ghostty (e.g. Neovim, tmux, Zellij, etc.), please provide a minimum version of the relevant configuration information here.  If you use custom CSS or shaders for Ghostty, also include them here, if applicable to your issue.
```

I'd say it would be desirable to be verbose here, to avoid any misunderstanding that may occur.
```suggestion
        If your issue involves other programs, tools, or applications in addition to Ghostty (e.g. Neovim, tmux, Zellij, etc.), please provide the minimum configuration needed for all relevant programs to reproduce the issue here.  If you use custom CSS or shaders for Ghostty, also include them here, if applicable to your issue.
```